### PR TITLE
alpine based attempt lacking memcached sasl support

### PR DESCRIPTION
**Pull request by thematthopkins**

### CODE REVIEW

**Comment by thematthopkins** [comment]
This guy may be a good candidate if we decide to go w/o memcached sasl.  Not sure yet whether this solves any of the random memcached issues or ctrl+c issues we've seen though.

Feel free to close PR at any time - no action required.  Just for awareness.

File: Dockerfile [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+    wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-php5-memcached/master/sgerrand.rsa.pub && \
+    wget https://github.com/sgerrand/alpine-pkg-php5-memcached/releases/download/2.2.0-r0/php5-memcached-2.2.0-r0.apk && \
+    apk add --allow-untrusted --no-cache ./php5-memcached-2.2.0-r0.apk && \
+    rm php5-memcached-2.2.0-r0.apk && \
+    rm -rf /var/cache/apk/*
+
+RUN groupadd www-data && \
+    useradd -M -u 1000 -o www-data 
+
+RUN sed -i 's|^;date.timezone =.*|date.timezone = America/New_York|' /etc/php5/php.ini && \
```
Comment (jonpitch) [inline Dockerfile:27]: @thematthopkins we can bring these in from `php.ini`?

File: Dockerfile [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
-
-RUN php5enmod mcrypt
+    php5-pdo_pgsql \
+    php5-xdebug \ 
+    shadow \
+    openssl
+
+RUN apk --no-cache add ca-certificates && \
+    wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-php5-memcached/master/sgerrand.rsa.pub && \
+    wget https://github.com/sgerrand/alpine-pkg-php5-memcached/releases/download/2.2.0-r0/php5-memcached-2.2.0-r0.apk && \
```
Comment (jonpitch) [inline Dockerfile:19]: @thematthopkins What if we grabbed the source, `./configure --enable-sasl`, `make` and `install`?

**Review by jonpitch — CHANGES_REQUESTED** [review]
@thematthopkins What is the size of the resulting image?

Also, I think we want to keep SASL, but that's not my decision; I'd rather the team be ok with removing it.

File: Dockerfile (hunk 2 above)
Comment (thematthopkins) [reply Dockerfile:19]: We can do that, but then we have to set this image up w/ gcc, php-dev, etc.  That may be the way to go though.  I kind of feel like we're heading towards the php image approach then too, so maybe the php image is the better approach?

File: Dockerfile (hunk 1 above)
Comment (thematthopkins) [reply Dockerfile:27]: I found the the php-fpm confs didn't override pools the way that that you's think, so I didn't trust the ini files to do the right thing either.  I think you're right though, we could should be able to pull these out into a separate .ini file.  It's kind of nice having a single php.ini as the source of truth though too.

**Comment by thematthopkins** [comment]
@jonpitch - the resulting image is 54 MB vs the 266 MB for the swimsuit season branch, or 532 MB for ubuntu.

**Pull request closed without merging**